lib/cadet/assessments/assessments.ex

```diff
@@ -500,6 +501,35 @@ defmodule Cadet.Assessments do
     Question.changeset(%Question{}, params_with_assessment_id)
   end
 
+  def update_final_contest_entries do
+    # 1435 = 1 day - 5 minutes
+    if Log.log_execution("update_final_contest_entries", Duration.from_minutes(1435)) do
+      Logger.info("Started update of contest entry pools")
+      questions = fetch_unassigned_voting_questions()
+
+      for q <- questions do
+        insert_voting(q.course_id, q.question["contest_number"], q.question_id)
+      end
```

Comment:
Question: Should the return value of this function be checked here? What happens if this line throws an error?

E.g. what if when the voting entry was originally inserted, the number of the contest/assessment in question still exists, but is sometime later deleted? This line would fail as a result, right?

Reply:
As discussed offline, deleting the assessment will delete the question, so the for loop would be empty, so no error will be thrown.

Thanks!

Reply:
Update: Deleting the assessment will delete the assessment question, but not the voting question (which is the one we are interested in!). #1002 will resolve this by enforcing a precondition for this loop.